src/components/catalog/selectable/selectable.stache

```diff
@@ -19,9 +19,7 @@ limitations under the License.
 
 <div class="selectable-container {{containerClasses}}" ($click)="onSelect()">
   <div class="logo-container">
-    {{#if brandLogoURL}}
-      <img src="{{brandLogoURL}}" title="{{thingName}}"/>
-    {{/if}}
+    <span>{{thingName}}</span>
```

Comment:
looks like this was intended to be part of #11, instead of 12?

Reply:
Yes, that is the change intended for #11.  I think I accidentally overwrote the previous message with my second request, because that's the one with the fix for account creation